gene_gene/gene_interacts_gene/dataset_statistics/dataset_statistics.py

```diff
@@ -0,0 +1,520 @@
+
+# coding: utf-8
+
+# # Dataset Statistics for Gene Gene Sentences
+
+# This notebook is designed to show statistics on the data extracted from pubmed. The following cells below here are needed to set up the environment.
+
+# In[1]:
+
+
+get_ipython().run_line_magic('load_ext', 'autoreload')
+get_ipython().run_line_magic('autoreload', '2')
+get_ipython().run_line_magic('matplotlib', 'inline')
+
+from collections import Counter
+from itertools import product
+import os
+import pickle
+import sys
+
+sys.path.append(os.path.abspath('../../../modules'))
+
+import pandas as pd
+import matplotlib.pyplot as plt
+from matplotlib_venn import venn2
+import seaborn as sns
+from tqdm import tqdm_notebook
+
+sns.set(rc={'figure.figsize':(12,6), "font.size":17})
+
+
+# In[2]:
+
+
+#Set up the environment
+username = "danich1"
+password = "snorkel"
+dbname = "pubmeddb"
+
+#Path subject to change for different os
+database_str = "postgresql+psycopg2://{}:{}@/{}?host=/var/run/postgresql".format(username, password, dbname)
+os.environ['SNORKELDB'] = database_str
+
+from snorkel import SnorkelSession
+session = SnorkelSession()
+
+
+# In[3]:
+
+
+from snorkel.models import candidate_subclass, Candidate
+GeneGene = candidate_subclass('GeneGene', ['Gene1', 'Gene2'])
+
+
+# In[4]:
+
+
+from utils.notebook_utils.dataframe_helper import write_candidates_to_excel, make_sentence_df
+
+
+# # Read Full Sentence Table
+
+# The cells below will read every sentence that contains a gene and disease entity from the sentence table in our postgres database. For time sake majority of the data has already been processed and save as files mentioned below.
```

Comment:
"gene and gene entity"?

Reply:
Is this looking for sentences with 2 gene's mentioned?  Or <gene><word><gene> pattern?

Reply:
Looking at sentences with 2 or more gene entities.